nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexSurefireExecution.java

```diff
@@ -79,9 +82,11 @@ protected void setupArgline() {
             configElement.getChild("test").setValue(this.configuration.testName);
         }
         Logger.getGlobal().log(Level.FINE, "Running surefire with: " + this.configuration.toArgLine());
+        Logger.getGlobal().log(Level.SEVERE, mavenRepoLocal);
```

Comment:
we don't need this log. Definitely not at Level.SEVERE. You may add it with Level.FINER; if you do, add a message saying what it is logging.